Widget/Settings/WidgetProfile.cpp

```diff
@@ -42,7 +43,7 @@ WidgetProfile::WidgetProfile() : Glib::ObjectBase("WidgetProfile") {
     grid->attach(m_status, 2, 1, 1, 1);
 
     grid->attach(*Gtk::manage(new Gtk::Label("Tox ID", 1, 0.5)), 1, 2, 1, 1);
-    auto tox_id = Gtk::manage(new Gtk::Label("AABBCCDDEEFFF...", 0, 0.5));
+    auto tox_id = Gtk::manage(new Gtk::Label(Tox::to_hex(Tox::instance().get_address().data(), 32), 0, 0.5));
```

Comment:
In this case it's not 32 bytes long !
It's `Tox::instance().get_address().data().size()` bytes long